pkg/apiserver/event/event.go

```diff
@@ -136,3 +139,42 @@ func (s *Service) listDryEvents(c *gin.Context) {
 
 	c.JSON(http.StatusOK, eventList)
 }
+
+// @Summary Get the event from db by ID.
+// @Description Get the event from db by ID.
+// @Tags events
+// @Produce json
+// @Param id query uint true "The id of the event"
+// @Success 200 {object} core.Event
+// @Router /events/get [get]
+// @Failure 500 {object} utils.APIError
+func (s *Service) getEvent(c *gin.Context) {
+	idStr := c.Query("id")
+
+	if idStr == "" {
+		c.Status(http.StatusBadRequest)
+		_ = c.Error(utils.ErrInvalidRequest.New("id cannot be empty"))
+		return
+	}
+
+	id, err := strconv.Atoi(idStr)
+	if err != nil {
+		c.Status(http.StatusBadRequest)
+		_ = c.Error(utils.ErrInvalidRequest.New("the format of id is wrong"))
+		return
+	}
+
+	event, err := s.event.Find(context.Background(), uint(id))
+	if err != nil {
+		if !gorm.IsRecordNotFoundError(err) {
+			c.Status(http.StatusInternalServerError)
+			_ = c.Error(utils.ErrInternalServer.NewWithNoMessage())
```

Comment:
Why not returning this error?

Reply:
This is consistent with "/archives/detail".
If we want to modify it, I will also change "/archives/detail"?